src/main/java/jp/ne/paypay/model/Payment.java

```diff
@@ -289,7 +296,31 @@ public String getAmountDescription() {
   public void setAmountDescription(String amountDescription) {
     this.amountDescription = amountDescription;
   }
-  
+
+
+  public String getPaymentMethodType() {
+    return paymentMethodType;
+  }
+
+  public String getPaymentMethodId() {
+    return paymentMethodId;
+  }
+
+  public void setPaymentMethod(PaymentMethod paymentMethod) {
+    this.paymentMethodType = paymentMethod.getPaymentMethodType();
+    this.paymentMethodId = paymentMethod.getPaymentMethodId();
+  }
```

Comment:
[design explanation]
Providing setters for both `setPaymentMethodType` and `setPaymentMethodId` is a reasonable choice, but here I provided a single setter that accepts `PaymentMethod` instance instead.

The reason for this implementation is to make it easier for users to use sdk. Refer to the example usage I added in this PR at PaymentApi.md.

Glad to hear any comment.

Reply:
Having `setPaymentMethod(PaymentMethod paymentMethod)` would be good idea. 
However I'd like to have setter method for paymentMethodId and paymentMethodType.

I assume you expect that merchants request getPaymentMethods every time but merchants might want to set paymentMethodId and paymentMethodType which was obtained before hand like values already stored in merchants side database.
Therefore I'd like to keep having setter methods for paymentMethodId and paymentMethodType to have flexibility.

Would you please consider this point?

note `PaymentMethod` class does not have setter as it just indicates api response now

Reply:
Makes sense to me.
I changed the implementation.